runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerItems.java

```diff
@@ -41,6 +41,9 @@
 	ANCIENT_BLESSING(ItemID.ANCIENT_BLESSING, 1),
 	HONOURABLE_BLESSING(ItemID.HONOURABLE_BLESSING, 1),
 	WAR_BLESSING(ItemID.WAR_BLESSING, 1),
+	RADAS_BLESSING_2(ItemId.RADAS_BLESSING_2, 1),

```

Comment:
```suggestion
	RADAS_BLESSING_2(ItemID.RADAS_BLESSING_2, 1),
```

Same for the other two lines.